commit 37807a694f89611f60880260d2bb7162908bc0c8
Author: Andrew Hughes <gnu.andrew@redhat.com>
Date:   Wed Mar 30 04:19:43 2022 +0100

    8283911: DEFAULT_PROMOTED_VERSION_PRE not reset to 'ea' for jdk-17.0.4

diff --git openjdk.orig/make/conf/version-numbers.conf openjdk/make/conf/version-numbers.conf
index 71b19762f2e..7378ec67a48 100644
--- openjdk.orig/make/conf/version-numbers.conf
+++ openjdk/make/conf/version-numbers.conf
@@ -39,4 +39,4 @@ DEFAULT_VERSION_CLASSFILE_MINOR=0
 DEFAULT_VERSION_DOCS_API_SINCE=11
 DEFAULT_ACCEPTABLE_BOOT_VERSIONS="16 17"
 DEFAULT_JDK_SOURCE_TARGET_VERSION=17
-DEFAULT_PROMOTED_VERSION_PRE=
+DEFAULT_PROMOTED_VERSION_PRE=ea
